src/main/java/org/cactoos/text/BytesAsText.java

```diff
@@ -114,4 +114,9 @@ public String asString() throws IOException {
         return new String(this.bytes.asBytes(), this.charset);
     }
 
+    @Override
+    public int compareTo(final Text text) {
+        return new UncheckedText(text).compareTo(this);
```

Comment:
@fabriciofx I understand that it will lead to replicate the code, but when I read 

> returns a negative integer, zero, or a positive integer as this object is less than, equal to, or greater than the specified object.

 I wait that the result will be same as in this contract or at least the same for all classes include `UncheckedText`